examples/fineweb_edu_classifier_example.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Should move this into `examples/classifiers` since https://github.com/NVIDIA/NeMo-Curator/pull/172 was merged.

Reply:
Yes, we should.

Reply:
Resolved